client/server/render/index.js

```diff
@@ -67,6 +67,8 @@ export function renderJsx( view, props ) {
 	return doctype + ReactDomServer.renderToStaticMarkup( createElement( component, props ) );
 }
 
+// JSdoc rule can't handle the function throwing an Error instead of returning.
```

Comment:
Can't we just use `@throws`?

Reply:
Hm, so that didn't work when I originally tried it, but thanks for asking me to take a second look. It appears that JSDoc was correct -- this function can return `undefined sometimes, because this is the catch statement:

https://github.com/Automattic/wp-calypso/blob/beef4ae07d0b4d825bd79f6edf73fdc55e378048/client/server/render/index.js#L137-L147

So we need to update the return type instead